tornado/websocket.py

```diff
@@ -181,6 +184,7 @@ def get(self, *args, **kwargs):
 
         self.ws_connection = self.get_websocket_protocol()
         if self.ws_connection:
+            self.clear_header('Content-Type')
```

Comment:
Clearing `Content-Type` (and only `Content-Type`) is fishy. Does it matter to browsers whether `Content-Type` is set on websocket handshake responses? I'd rather let a bogus `Content-Type` through here than treat it as a special case as long as it works.